setup.py

```diff
@@ -0,0 +1,19 @@
+"""
+This is a setup.py script generated by py2applet
+
+Usage:
+    python setup.py py2app
+"""
+
+from setuptools import setup
+
+APP = ['minesweeper.py']
+DATA_FILES = ['doomguy.gif', 'bomb_small.png', 'bomb.png', 'button_small.png', 'flag_small.png', 'pressed_button_small.png']
```

Comment:
This file needs blacking. Change black command in .github/workflows/main.yml from `python3 -m black --check minesweeper.py` to `python3 -m black --check minesweeper.py setup.py`